Makefile

```
@@ -14,7 +14,7 @@ clean:
 .PHONY: install # install in a virtualenv
 install:
 	python3 -m venv .venv && source .venv/bin/activate && \
-	pip3 install wheel 'sphinx<7' && \
+	pip3 install wheel 'sphinx==7.1.0' && \
 	python3 setup.py install
```

See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html; you should prefer using standards-based toolchains. That may resolve the installation problem.

```suggestion
	python3 -m pip install .
```

---

@AA-Turner thank you!
I have also verified that it works by installing the package through `pip3 install --force git+https://github.com/nodiscc/hecat.git@upgrade-sphinx`. This indeed appears to fix https://github.com/sphinx-doc/sphinx/issues/11130. I will update the issue with this information.